.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor/
-composer.lock
```

Comment:
Why include the composer.lock file in a library?

Reply:
> # Lock file
For your library you may commit the composer.lock file if you want to. This can help your team to always test against the same dependency versions. However, this lock file will not have any effect on other projects that depend on it. It only has an effect on the main project.

https://getcomposer.org/doc/02-libraries.md#lock-file

It does not interfere with other projects that will come from this library but make sure the tests on CI are run on the same versions as the library was tested by developer.

It also is required for caching dependencies by GHA.